controllers/shipwrightbuild_controller.go

```diff
@@ -233,6 +252,16 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		err = r.Client.Status().Update(ctx, b)
 		return RequeueWithError(err)
 	}
+
+	err = deleteObjectsIfPresent(ctx, r.Client, []client.Object{
```

Comment:
We should have comment here explaining why this code exists:

```suggestion
	// Builds 0.12.0 created a ClusterRole and ClusterRolebinding for the Build API conversion webhook.
	// These were removed in v0.13.0 - when upgrading, these should be removed if present.
	err = deleteObjectsIfPresent(ctx, r.Client, []client.Object{
```